internal/commands/push/command.go

```diff
@@ -153,23 +153,25 @@ func (cmd *Command) Handler(profile *cli.Profile, ui terminal.UI, clients cli.Cl
 		return nil
 	}
 
-	ui.Print(terminal.NewTextLog("Creating draft"))
-	draft, proceed, err := createNewDraft(ui, clients.Realm, to)
-	if err != nil {
-		return err
-	}
-	if !proceed {
-		return nil
-	}
+	if len(appDiffs) > 0 {
```

Comment:
driveby fix so we don't go through the draft/import/deploy cycle unless we need to